traffic_ops/traffic_ops_golang/federations/allfederations.go

```diff
@@ -125,13 +115,15 @@ FROM
 ORDER BY
   ds.xml_id
 `
-	imsQuery := `SELECT max(t) from (
-		SELECT max(fds.last_updated) as t FROM
-  federation_deliveryservice fds
-  JOIN deliveryservice ds ON ds.id = fds.deliveryservice
-  JOIN federation fd ON fd.id = fds.federation
-	UNION ALL
-	select max(last_updated) as t from last_deleted l where l.table_name='federation_deliveryservice') as res`
+	imsQuery := `SELECT max(last_updated) FROM (SELECT last_updated FROM federation_deliveryservice fds
+        UNION ALL SELECT last_updated FROM federation_federation_resolver ffr
+        UNION ALL SELECT last_updated FROM federation fd
+		UNION ALL
+    		SELECT max(last_updated) AS t FROM last_deleted l 
+			WHERE l.table_name='federation_deliveryservice' 
```

Comment:
nit: this can be `WHERE l.table_name IN ('federation_deliveryservice', 'federation', 'federation_federation_resolver')`

nit: we should run this through a SQL formatter so it's easier to read (maybe https://www.dpriver.com/pp/sqlformat.htm?)